homeassistant/components/samsungtv/__init__.py

```diff
@@ -45,12 +43,8 @@ async def async_setup(hass, config):
     if DOMAIN in config:
         hass.data[DOMAIN] = {}
```

Comment:
It would be better if we always set domain in hass.data so we don't need to check that when setting up the media player platform.